packages/cardpay-sdk/sdk/constants.ts

```diff
@@ -47,6 +48,7 @@ interface SchedulerCapableNetworkConstants {
   tokenList: TokenList;
   scheduledPaymentFeeFixedUSD: number;
   scheduledPaymentFeePercentage: number;
+  publicRpcUrls: string[];
```

Comment:
Needed for the case when the user doesn't have this network added in metamask yet, and we try to add it